docs/topics/testing/tools.txt

```diff
@@ -125,6 +125,11 @@ Use the ``django.test.Client`` class to make requests.
     The ``enforce_csrf_checks`` argument can be used to test CSRF
     protection (see above).
 
+    The ``json_encoder`` argument allows you to set a custom JSON encoder.
+    This will be used when providing JSON data as a dictionary to either
```

Comment:
The requirement to specify content_type=... should be mentioned.

Reply:
This is kind of stated in the paragraph above. But re-iterating anyway.

Reply:
Done.

Reply:
Ah wait I missed the context of this. I will add.

Reply:
_now done_